server/src/test/java/io/druid/client/CachingClusteredClientTest.java

```diff
@@ -3087,7 +3102,10 @@ public void testIfNoneMatch()
     TimeBoundaryQuery query = Druids.newTimeBoundaryQueryBuilder()
                                     .dataSource(DATA_SOURCE)
                                     .intervals(new MultipleIntervalSegmentSpec(ImmutableList.of(interval)))
-                                    .context(ImmutableMap.<String, Object>of("If-None-Match", "aVJV29CJY93rszVW/QBy0arWZo0="))
+                                    .context(ImmutableMap.<String, Object>of(
```

Comment:
`ImmutableMap.<String, Object>of` can be `ImmutableMap.of`

Reply:
that one was caught in a source code refactor I think :) I'll fix

Reply:
Fixed